### DISCUSSION TITLE
Offical interface is available?

### DISCUSSION THREAD

**Discussion by Microbion** [nextflow-io/nextflow#4012] [Q&A]
I found NF-web described [here](https://www.nextflow.io/nfcamp/2019/anton.html). But I didn't know how to use it. Is it available? Re-mention #88. From now on, I found three third-party suports: [NextflowWorkbench](https://github.com/manuelesimi/NextflowWorkbench), [dolphinnext](https://github.com/UMMS-Biocore/dolphinnext) and [nf-cloud](https://github.com/mpc-bioinformatics/nf-cloud).
And how about building an jupyter extension to trace nextflow tasks(like ipyparallel)?

**Comment by pditommaso** [comment]
That's an old project (2019). The "official" UI for nextflow is Tower https://cloud.tower.nf/

**Reply by Microbion** [reply]
Got it. Thanks a lot!
